edgelet/edgelet-core/src/settings.rs

```diff
@@ -442,6 +442,13 @@ impl Listen {
 
 #[derive(Clone, Debug, serde_derive::Deserialize, serde_derive::Serialize)]
 pub struct Certificates {
+    #[serde(flatten)]
+    device_cert: Option<DeviceCertificate>,
+    auto_generated_ca_lifetime: u16,
```

Comment:
Should this be u64? But then below you multiply this by 86,400... so you should check overflow.

Reply:
Discussed offline and we decided that u16 is fine since the input from the config.yaml is in days.